### DISCUSSION TITLE
Unfortunately, File Explorer is crashing :(

### DISCUSSION THREAD

**Discussion by PGrecoo** [valinet/ExplorerPatcher#1575] [General]
Hi all,
I have a problem with EP. I have installed the latest version but at one point it crashed so I decided to unistall following the [Wiki](https://github.com/valinet/ExplorerPatcher/wiki) but now I have the following problem and I cannot restore the default properties like the start menu modified.

I can unistall EP by following the wiki but the option I used, they remain.
How can I fix that? I cannot restore the default properties.

![unfortunately](https://github.com/valinet/ExplorerPatcher/assets/37579392/f5a2073f-84c1-4836-9141-63b503e7bb96)

Thank you for support.

**Comment by pyrates999** [comment]
Please post your windows version and build, you can find it by running winver.  Google to be sure it is not a preview or insider build as those are not supported due to Microsoft not supply symbols to download for that.
Please post the explorer patcher version you installed if you remember it.

Then try the following:

1. uninstall explorer patcher
2. delete the same registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. install explorer patcher
4. verify you get a notification that symbols have been downloaded
5. restart windows explorer using either the task manager or in the properties window of explorer patcher
6. Reapply your settings in explorer patcher

**Comment by PGrecoo** [comment]
Hi again,
thank you for supporting me. I will post the version 

![Screenshot 2023-06-21 142307](https://github.com/valinet/ExplorerPatcher/assets/37579392/c2444937-8a90-4459-b138-2e436abfd8f8)

I will try this procedure. Thank you again

**Comment by PGrecoo** [comment]
Well, I have tried as you described me, but the only thing that is still there is the w11 context menu.

I'm trying to disable this option now :) 

Thank you for answering me

EDIT: 
Probably I fixed, giving permission and deleting the following reg key {86ca1aa0-34aa-4e8b-a509-50c905bae2a2} created by EP.

Now can you tell me if there will be any problem with my W11 distrib?
I want to use EP but when it works properly for my w11 edition. Thank you again for supporting me
